.github/workflows/domain_ci.yml

```diff
@@ -36,14 +40,19 @@ jobs:
       - name: Install torchdata
         run: python setup.py install
 
-      - name: Install test utilities
-        run: pip install pytest pytest-mock
+      - name: Install test requirements
+        run: pip install pytest pytest-mock scipy iopath
+
+      - name: Extract torchvision ref
+        id: torchvision
+        run: echo "::set-output name=ref::$(python -c 'import torchvision; print(torchvision.version.git_version)')"
```

Comment:
This makes sure that we always run the tests of the same commit the nightly version was build for. Otherwise we might see failures here that have nothing to do with `torchdata`.

Reply:
This is awesome!